data-serving/data-service/test/case.test.ts

```diff
@@ -49,35 +49,48 @@ describe('POST', () => {
     it('create without valid date should return 422', (done) => {
         request(app)
             .post('/api/cases')
-            .send({ date: 'not-a-date', outcome: 'recovered' })
+            .send({
+                eventSequence: { confirmed: 'not-a-date' },
+                outcome: 'Recovered',
+            })
             .expect(422, done);
     });
     it('create without valid outcome should return 422', (done) => {
         request(app)
             .post('/api/cases')
-            .send({ date: '2020-04-01', outcome: 'not-a-valid-outcome' })
+            .send({
+                eventSequence: { confirmed: new Date('2020-04-01') },
+                outcome: 'not-a-valid-outcome',
+            })
             .expect(422, done);
     });
     it('create with valid input should return 200 OK', async () => {
         const inputDate = new Date('2020-04-01').toJSON();
-        const inputOutcome = 'recovered';
+        const inputOutcome = 'Recovered';
         const res = await request(app)
             .post('/api/cases')
-            .send({ date: inputDate, outcome: inputOutcome })
+            .send({
+                eventSequence: { confirmed: inputDate },
+                outcome: inputOutcome,
+            })
             .expect('Content-Type', /json/)
             .expect(200);
+        console.log('\n\n' + res.body.toString());
```

Comment:
My precious console lines...

Reply:
EEK you found me out. still having trouble getting breakpoints to work with vscode + npm run test